oarepo_ui/resources/templating/filters.py

```diff
@@ -63,6 +63,29 @@ def field_value(value):
     return value
 
 
+def field_data(value, ui=None):
+    if isinstance(value, FieldData):
+        return value
+    return FieldData(value, ui=ui or {})
+
+
+def field_get(value, key):
+    print(f"field_get({key})")
+    if isinstance(key, FieldData):
+        key = field_value(key)
+    print("key", key)
+    if key is None:
+        return value
+
+    if not isinstance(value, FieldData):
```

Comment:
This looks like field_data()

Reply:
@mirekys @mesemus Hey, I've made the changes according to how I understood the issue. The normal styling of the custom fields in detail page, looks OK to me and so I was not sure what if anything to include in some of these less components. For now, their classes are simply attached to their respective parts of the Icustomfields, and so that various levels of the component can be easily overriden, which I hope is the goal :) Please take a look